features/admin.applications.v1/components/forms/advanced-configurations-form.tsx

```diff
@@ -362,195 +431,337 @@ export const AdvancedConfigurationsForm: FunctionComponent<AdvancedConfiguration
+                                        data-componentid={ `${ testId }-enable-api-based-authentication` }
+                                        disabled={ isClientAttestationUIDisabled }
+                                        hint={ t("applications:forms.advancedConfig." +
+                                            "sections.clientAttestation.fields." +
+                                            "enableClientAttestation.hint") }
+                                    />
+                                </Grid.Column>
+                            </Grid.Row>
+                            <Grid.Row
+                                columns={ 1 }
+                                onClick={ handleWhenClientAttestationClickedWhenDisabled }>
+                                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                                    <Field.Textarea
+                                        ariaLabel="Android service account credentials"
+                                        inputType="description"
+                                        name="androidAttestationServiceCredentials"
+                                        label={ t("applications:forms." +
+                                            "advancedConfig." +
+                                            "sections.clientAttestation.fields." +
+                                            "androidAttestationServiceCredentials.label") }
+                                        placeholder={ t("applications:forms." +
+                                            "advancedConfig.sections." +
+                                            "clientAttestation.fields." +
+                                            "androidAttestationServiceCredentials.placeholder") }
+                                        value={
+                                            config?.
+                                                attestationMetaData?.androidAttestationServiceCredentials ?
+                                                [ JSON.stringify(
+                                                    config?.
+                                                        attestationMetaData?.
+                                                        androidAttestationServiceCredentials,
+                                                    null, 4)
+                                                ] : []
+                                        }
+                                        hint={ t("applications:forms." +
+                                            "advancedConfig." +
+                                            "sections.clientAttestation.fields." +
+                                            "androidAttestationServiceCredentials.hint") }
+                                        type="text"
+                                        maxLength={ 5000 }
+                                        minLength={ 30 }
```

Comment:
put these values into a const file. Do not hard code.

Reply:
This is an existing code block which was refactored.